app/lib/validator.py

```diff
@@ -29,6 +29,7 @@ def validate_request(request, schema):
                 raise Exception(f"{property} doesn't exsist in the schema!")
```

Comment:
For the time being please comment out validation for properties in node. There needs to be some updates to be done in the schema

Reply:
also if 'properties' field is not present handling it by adding empty object would be better
```
if 'properties' not in node:
            raise Exception("properties is required")
```

Reply:
done